mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -138,15 +132,14 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
                                           startDate=startDateFirstYear,
                                           endDate=endDateFirstYear,
                                           calendar=calendar)
-        dsFirstYear = xr.open_mfdataset(
-            filesFirstYear,
-            preprocess=lambda x: preprocess_mpas(x,
-                                                 yearoffset=yearOffset,
-                                                 timestr='Time',
-                                                 onlyvars=variableList,
-                                                 varmap=variableMap))
-
-        dsFirstYear = remove_repeated_time_index(dsFirstYear)
+        dsFirstYear = openMultifileDataSet(filesFirstYear,
+                                           calendar,
```

Comment:
minor comment, but shouldn't the two lines above have inFiles=filesFirstYear and calendar=calendar, for style consistency?